src/Mailer/Email.php

```diff
@@ -1023,7 +1023,7 @@ protected function _constructTransport($name)
             return $config['className'];
         }
 
-        $className = App::className($config['className'], 'Network/Email', 'Transport');
+        $className = App::className($config['className'], 'Mailer/Transport', 'Transport');
```

Comment:
We should check the new and then old namespaces. Otherwise userland code will break.